scripts/bump-spin-formula.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+VERSION=$1
+FORMULA=${2:-Formula/Spin.rb}
+
+usage() {
+    echo "Usage: $0 <VERSION> [<FORMULA_PATH>]"
+    echo "Updates the Spin Formula for the specified release version"
+    echo "Example: $0 3.0.0"
```

Comment:
Should this example text be updated to `v3.0.0` in line with the logic of this script?

Tangent: I see in the Spin formula that we utilize `v#{version}` for all the spots where the version is used (eg https://github.com/fermyon/homebrew-tap/blob/main/Formula/spin.rb#L7).  Is there something holding us back from just updating it to `#{version}` and then updating this script to not have a need to chop the leading 'v'?

Reply:
Good call out on the example! I haven't seen it documented, but there is a clear convention that the top level version of the file `version "2.3.1"` should not be prefixed with a `v` . Every formula in [homebrew-core](https://github.com/Homebrew/homebrew-core) follows this. Though `brew test-bot --only-tap-syntax Formula/spin.rb ` passes when i prefix with a `v`